python/ml4ir/base/features/feature_fns/categorical.py

```diff
@@ -444,7 +443,8 @@ def categorical_indices_from_vocabulary_file(feature_info, feature_tensor, file_
         feature_name=feature_info.get("node_name", feature_info["name"]),
     )
     categorical_indices = lookup_table(feature_tensor)
-    return categorical_indices, vocabulary_keys, num_oov_buckets
+
+    return categorical_indices, vocabulary_size, num_oov_buckets
```

Comment:
I refactored to return vocabulary_size. vocabulary_keys was used by callers only for computing the vocabulary size

Reply:
Nice.